code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -30,18 +30,25 @@
 	if(feels_pain() && !has_painkillers())
 		if(pain_shock_stage >= 50)
 			. += 3
-
-		for(var/organ_name in list(LIMB_LEFT_FOOT,LIMB_RIGHT_FOOT,LIMB_LEFT_LEG,LIMB_RIGHT_LEG))
+		var/list/limbs_to_check
+		var/multiplier = 1
+		if(!lying)
+			limbs_to_check = list(LIMB_LEFT_FOOT,LIMB_RIGHT_FOOT,LIMB_LEFT_LEG,LIMB_RIGHT_LEG)
+		else
+			limbs_to_check = list(LIMB_LEFT_ARM,LIMB_RIGHT_ARM,LIMB_LEFT_HAND,LIMB_RIGHT_HAND)
```

Comment:
I believe there's a list called grasp_organs, which contains all arms+hands. Use it instead